### ISSUE TITLE
Simple change massive increase

### ISSUE THREAD

**Issue by vegansquirrel** [vukrosic/ultimate-ai-research-and-engineering-course#1]
Hi, I tried the mod_god_speedrun . It has been a while since I worked on MLP.

For the first attempt I simply increased the batch size from 128 to 256. and these were the results.

🏆 SPEEDRUN SUMMARY:
Success: YES
Final validation accuracy: 95.70%
Total time: 21.35 seconds
Epochs completed: 162
Test accuracy: 95.90%

🎉 CONGRATULATIONS! NEW RECORD SET! 🎉
🏆 Previous record: 23.42s
🏆 Your new record: 21.35s
📊 Improvement: 2.07s faster!

💡 Remember to update CURRENT_RECORD_TIME in Cell 1!

------------

I also tried to increase it to 512 but that did not give the required accuracy in the given 170 Epochs. 
However it did reduce time to 18 sec.

So then I tried , increasing learning rate to 0.005 and epochs to 200, with 512 batch size and this is the result.

🏆 SPEEDRUN SUMMARY:
Success: YES
Final validation accuracy: 95.05%
Total time: 9.85 seconds
Epochs completed: 96
Test accuracy: 95.00%

🎉 CONGRATULATIONS! NEW RECORD SET! 🎉
🏆 Previous record: 23.42s
🏆 Your new record: 9.85s
📊 Improvement: 13.57s faster!

💡 Remember to update CURRENT_RECORD_TIME in Cell 1!

This was a huge leap.

**Comment by vegansquirrel** [comment]
And modifying a bit more , with a learning rate of 0.03 and batch size of 512 we get better accuracy as well as time.

🏆 SPEEDRUN SUMMARY:
Success: YES
Final validation accuracy: 95.70%
Total time: 9.21 seconds
Epochs completed: 87
Test accuracy: 96.15%

🎉 CONGRATULATIONS! NEW RECORD SET! 🎉
🏆 Previous record: 23.42s
🏆 Your new record: 9.21s
📊 Improvement: 14.21s faster!

💡 Remember to update CURRENT_RECORD_TIME in Cell 1!
